auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -168,6 +180,17 @@ private AuthUI(FirebaseApp app) {
         mAuth.useAppLanguage();
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    public static void setApplicationContext(@NonNull Context context) {
```

Comment:
This is a tad bit ugly since we need our own `ContentProvider` to make it work, but it's needed to create the `GoogleSignInOptions` when retrieving an xml string. Then again, it could also be seen as a plus since the infrastructure will be there if we want to expand our magic.